ush/config_defaults.yaml

```diff
@@ -2901,8 +2901,10 @@ cpl_aqm_parm:
   # DO_AQM_GEFS_LBCS:
   # Add GEFS aerosol LBCs to chemical LBCs
   #
-  # DO_AQM_SAVE_AIRNOW_HIST:
-  # Save bias-correction airnow training data
+  # COMINbicor 
+  # Read bias-correction training data
+  # COMOUTbicor 
+  # Save bias-correction training data
```

Comment:
These descriptions for COMINbicor/COMOUTbicor are duplicated with the below (Ln 3004-3008). Please remove these.